client/api/todos.ts

```diff
@@ -0,0 +1,36 @@
+import type { InsertTodo, UpdateTodo } from "@/shared/todoSchema";
+
+export const fetchTodos = async () => {
```

Comment:
we need to zod validate the json we get back here (its typed unknown right now which is not very useful)